app/org/sagebionetworks/bridge/services/StudyConsentServiceImpl.java

```diff
@@ -134,4 +138,13 @@ private String loadDocumentContent(StudyConsent consent) {
         }
     }
     
+    private String sanitizeHTML(String documentContent) {
+        documentContent = Jsoup.clean(documentContent, Whitelist.relaxed());
+        Document document = Jsoup.parseBodyFragment(documentContent);
+        document.outputSettings().escapeMode(EscapeMode.xhtml);
```

Comment:
Just curious if the methods you call, like escapeMode(), return settings so that you can chain them together.